galsim/phase_screens.py

```diff
@@ -41,7 +41,7 @@ class AtmosphericScreen(object):
     @param altitude      Altitude of phase screen in km.  This is with respect to the telescope, not
                          sea-level.  [default: 0.0]
     @param time_step     Interval to use when advancing the screen in time in seconds.
-                         [default: 0.03]
+                         [default: 0.025]
```

Comment:
Alright.  Thanks for pointing this out, as it alerted me to another change I had forgotten to document.  Previously, setting the time_step was done only to the phase screens themselves.  With the new changes, there's still a time_step for the phase screens that's  used when boiling, but there's also a completely independent time_step used when creating the PSF.  I.e., boiling updates and PSF time integration steps no longer need to be synchronized, and in fact individual layers may have different boiling time_steps.

So I went through and documented that fact, and also added requirements that the screen time_step only be set (different from the new default of `None`) if boiling is turned on.

I did add something like your suggested text to the time_step documentation for makePSF and PhaseScreenPSF.